helion/_compiler/indexing_strategy.py

```diff
@@ -227,6 +227,9 @@ def valid_block_size(
         for i, k in enumerate(subscript):
             if k is None:
                 continue
+            if k is Ellipsis:
+                # Ellipsis is not supported in tensor descriptor mode
+                return False
```

Comment:
Is this hard to support?  Maybe we we pre-expand the indexing string the existing code could work unmodified.